osbs/conf.py

```diff
@@ -296,6 +296,9 @@ def get_distribution_scope(self):
     def get_architecture(self):
         return self._get_value("arch", self.conf_section, "architecture")
 
+    def get_platform(self):
+        return self._get_value("platform", self.conf_section, "platform")
```

Comment:
platform is more than CPU architecture. The names are likely to overlap but the concepts are different.

Yes, this would be better as a CLI argument.